docker/Dockerfile-synapsepy3

```diff
@@ -1,14 +1,35 @@
 FROM matrixdotorg/sytest:latest
```

Comment:
it'd be really nice to combine these two into one dockerfile with an ARG in the same way that synapse's dockerfile works.

Reply:
given that the lifespan of the py2 version is only a fortnight, not important, I guess.